annotations_models/train_random_forest_classifier_model.ipynb

```diff
@@ -46,31 +46,85 @@
     "from sklearn.ensemble import RandomForestClassifier\n",
     "from sklearn.preprocessing import StandardScaler\n",
     "import pprint\n",
+    "import os\n",
     "\n",
-    "from training.train import train_imbalanced_model, cross_validation_balance_on_date\n",
+    "from training.train import train_imbalanced_model, cross_validation_balance_on_date, create_folds\n",
     "from training.utils import get_cross_validation_results_filepath, get_model_filepath\n",
-    "from training.metric_calculation import calculate_average_metrics, get_metrics"
+    "from training.metric_calculation import calculate_average_metrics, get_metrics\n",
+    "import requests\n",
+    "\n",
+    "data_dir = os.path.abspath(os.path.join(os.getcwd(), '..')).replace(\"\\\\\",\"/\")+\"/data/annotations/annotations_pixel_dataframes/\"\n",
+    "\n",
+    "\n",
+    "def download_file(url, local_path):\n",
```

Comment:
I think it would be cleaner if this function was moved to an I/O source python file.

Reply:
good point for a further pr i would say